### ISSUE TITLE
False positives for terraform aws_eks_cluster resource if dynamic configuration is used

### ISSUE THREAD

**Issue by patrickpichler** [returntocorp/sgrep-rules#2475]
**Describe the bug**
The `terraform.lang.security.eks-insufficient-control-plane-logging.eks-insufficient-control-plane-logging` reports missing
configuration, if a dynamic block is present in the `aws_eks_cluster` resource.

**To Reproduce**
```
resource "aws_eks_cluster" "eks_cluster" {
  # ruleid: eks-insufficient-control-plane-logging
  name                      = "my-cluster-${var.test}"
  enabled_cluster_log_types = [
    "api",
    "audit",
    "authenticator",
    "controllerManager",
    "scheduler"
  ]

  dynamic "encryption_config" {
    for_each = 1
    content {
      provider {
        key_arn = aws_kms_key.k8s_cluster_secret_encryption_key.arn
      }
      resources = [
        "secrets"
      ]
    }
  }

}
```

**Expected behavior**
`eks_cluster` should not be flagged

**Priority**
How important is this to you?
- [ ] P0: blocking me from making progress
- [ ] P1: this will block me in the near future
- [x] P2: annoying but not blocking me

**Additional Context**
-

**Comment by colleend** [comment]
Hi! Thanks for filing this issue -- I've pushed a fix and after this is merged the rule should be good to go!
